dotCMS/src/main/java/com/dotmarketing/portlets/structure/action/EditFieldAction.java

```diff
@@ -318,6 +319,43 @@ private boolean _saveField(ActionForm form, ActionRequest req, ActionResponse re
                  }
             }
 
+            // Validate values entered for decimal/number type check box field
+            if (field.getFieldType().equals(FieldType.RADIO.toString())){
+                String values = fieldForm.getValues();
+                String temp = values.replaceAll("\r\n","|");
+                String[] tempVals = StringUtil.split(temp.trim(), "|");
+
+                if (tempVals.length % 2 == 1){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 2 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS109)